types/common.tsx

```diff
@@ -1,5 +1,6 @@
 // Type aliases
 export type I18nString = string
+export type ApiGatewayDateString = string
```

Comment:
Note: I added this alias so I could more easily see when I was expecting a string of the format `"yyyy-MM-dd'T'HH:mm:ss"`, which is the format that the API gateway presents datestrings as.